### DISCUSSION TITLE
Having issue when download .mp4 video on Android!

### DISCUSSION THREAD

**Discussion by tollar55** [prasathmani/tinyfilemanager#1115] [General]
When I download an .mp4 video through a Windows PC, everything is normal. But on Android phones (I've already tried it on 2 phones) the file changes from .mp4 to .mp3, for example I want to download video1.mp4 in the downloads folder it appears as video1.mp4.mp3 (become Audio format).
Does anyone also have this issue or is it of my phones?
![Screenshot_Download_1](https://github.com/prasathmani/tinyfilemanager/assets/83119245/b90c45de-50b8-483d-a5a1-5d5be1fe778c)
![Screenshot_Download_Result_2](https://github.com/prasathmani/tinyfilemanager/assets/83119245/c2ac007e-e558-47bb-b19e-08d01912bd14)

**Comment by prasathmani** [comment]
For now, use the direct link and save the file.
